From 8e12ea8fac0bc15e502c61ab9e2f57546bbe3091 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 15 Oct 2010 17:14:19 +0800
Subject: [PATCH] ENGR00132656 MX53 ARD: add MFG support

Add MFG tool support for MX53 ARD.
Remove MLB driver in updater defconfig because it
is not needed by MFG tool

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/configs/imx5_updater_defconfig |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx5_updater_defconfig b/arch/arm/configs/imx5_updater_defconfig
index b5422b5..60731b7 100644
--- a/arch/arm/configs/imx5_updater_defconfig
+++ b/arch/arm/configs/imx5_updater_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Sat Sep 25 14:12:27 2010
+# Fri Oct 15 17:09:24 2010
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -248,7 +248,7 @@ CONFIG_MX5_MULTI_ARCH=y
 CONFIG_MACH_MX51_3DS=y
 CONFIG_MACH_MX51_BABBAGE=y
 CONFIG_MACH_MX53_EVK=y
-# CONFIG_MACH_MX53_ARD is not set
+CONFIG_MACH_MX53_ARD=y
 CONFIG_MACH_MX50_ARM2=y
 CONFIG_MACH_MX50_RDP=y
 
@@ -829,6 +829,7 @@ CONFIG_APM_POWER=y
 # CONFIG_BATTERY_DS2782 is not set
 # CONFIG_BATTERY_BQ27x00 is not set
 # CONFIG_BATTERY_MAX17040 is not set
+# CONFIG_BATTERY_DS2438 is not set
 CONFIG_HWMON=y
 # CONFIG_HWMON_VID is not set
 # CONFIG_HWMON_DEBUG_CHIP is not set
@@ -1472,7 +1473,7 @@ CONFIG_GPS_IOCTRL=y
 #
 # MXC Media Local Bus Driver
 #
-CONFIG_MXC_MLB=y
+# CONFIG_MXC_MLB is not set
 
 #
 # i.MX ADC support
-- 
1.5.4.4

